src/cpp/_pythia8.cpp

```diff
@@ -209,6 +209,7 @@ PYBIND11_MODULE(_pythia8, m)
         .def("init", &Pythia::init, py::call_guard<py::scoped_ostream_redirect, py::scoped_estream_redirect>())
         .def("next", py::overload_cast<>(&Pythia::next))
         .def("readString", &Pythia::readString, "setting"_a, "warn"_a = true)
+        .def("forceHadronLevel", &Pythia::forceHadronLevel, "find_junctions"_a = true)
```

Comment:
I cannot check right now (on my phone): forceHadronLevel should support the same arguments as the C++ version. If that's the case, it is all good.

Reply:
I checked it once again. forceHadronLevel has only one argument. I use a default value.  It seems to work fine.